core/Pos/Binary/Core/Txp.hs

```diff
@@ -8,11 +8,13 @@ import           Universum
 
 import qualified Data.ByteString.Lazy as LBS
 
+import           Pos.Binary.Core.Common ()
+import           Pos.Binary.Core.Script ()
+import           Pos.Binary.Crypto ()
 import           Pos.Binary.Class (Bi (..), Cons (..), Field (..), decodeKnownCborDataItem,
                                    decodeListLenCanonical, decodeUnknownCborDataItem,
                                    deriveSimpleBi, encodeKnownCborDataItem, encodeListLen,
                                    encodeUnknownCborDataItem, enforceSize, matchSize)
-import           Pos.Binary.Core.Address ()
```

Comment:
This module was only imported so that *orphan instances* would be in scope... for *reexport*. Wow.